web/src/routes/events/Events.tsx

```diff
@@ -186,27 +186,31 @@ class Events extends React.Component<EventsProps, EventsState> {
                 <MqText heading>{i18next.t('events_route.title')}</MqText>
                 Page: {this.pageNavigation()}
               </Box>
-              <Box>
-                <Tooltip title={i18next.t('events_route.previous_page')}>
-                  <IconButton
-                    className={classes.ml2}
-                    color='primary'
-                    disabled={page === 1}
-                    onClick={() => this.handleClickPage('prev')}
-                  >
-                    <ChevronLeftRounded />
-                  </IconButton>
-                </Tooltip>
-                <Tooltip title={i18next.t('events_route.next_page')}>
-                  <IconButton
-                    color='primary'
-                    disabled={pageIsLast}
-                    onClick={() => this.handleClickPage('next')}
-                  >
-                    <ChevronRightRounded />
-                  </IconButton>
-                </Tooltip>
-              </Box>
+              {events.length === 0 ? (
+                <></>
+              ) : (
```

Comment:
I would use the `&&` notation here since you want to return a null value instead of an empty logical tag.

Reply:
@phixMe I've added a new commit addressing the change! Could you please have a quick review of it?